### ISSUE TITLE
[Bug]: Crash on startup NoClassDefFoundError/MineTogetherChat (6.0.7/6.0.8)

### ISSUE THREAD

**Issue by Raidobw2** [CreeperHost/CreeperHostGui#641]
### Describe the bug

For some players they seem to be getting a startup crash with a no class def error on the StoneBlock 3 modpack

### Steps to reproduce

1. Install FTB StoneBlock 3
2. Start the pack
3. Crashes

### Expected behaviour

That it would not crash on startup

### Screenshots

I don't have any, but I have logs, most of which are found in https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1597. Here's the recap:
### Crash
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 11/3/22, 9:49 AM
Description: Initializing game

java.lang.NoClassDefFoundError: Could not initialize class net.creeperhost.minetogether.chat.MineTogetherChat
	at net.minecraft.client.gui.Gui.handler$bac000$onInit(Gui.java:1381) ~[client-1.18.2-20220404.173914-srg.jar%23260!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:minetogether-common.mixins.json:chat.ingame.GuiMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.Gui.<init>(Gui.java:161) ~[client-1.18.2-20220404.173914-srg.jar%23260!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:minetogether-common.mixins.json:chat.ingame.GuiMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.gui.ForgeIngameGui.<init>(ForgeIngameGui.java:322) ~[forge-1.18.2-40.1.84-universal.jar%23265!/:?] {re:mixin,re:classloading,pl:epic_rei_lmao:B,pl:mixin:APP:minetogether-forge.mixins.json:ForgeIngameGuiMixin,pl:mixin:A}
	at net.minecraft.client.Minecraft.<init>(Minecraft.java:518) ~[client-1.18.2-20220404.173914-srg.jar%23260!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.json:MinecraftMixin,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:tklib.mixin.json:client.MinecraftMixin,pl:mixin:APP:botania_xplat.mixins.json:client.AccessorMinecraft,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:bookshelf.common.mixins.json:client.AccessorMinecraft,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:mixin.dynamic_asset_generator.json:MinecraftMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:169) ~[client-1.18.2-20220404.173914-srg.jar%23260!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.1.84.jar%2351!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:57) [ForgeWrapper-mmc3.jar:mmc3] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:214) [NewLaunch.jar:?] {}
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:249) [NewLaunch.jar:?] {}
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?] {}
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?] {}
```

### Latest.log
```
[04Nov2022 08:36:03.257] [modloading-worker-0/INFO] [net.creeperhost.minetogether.MineTogether/]: Initializing MineTogether!
[04Nov2022 08:36:03.548] [modloading-worker-0/INFO] [net.creeperhost.minetogether.MineTogetherClient/]: Initializing MineTogetherClient!
[04Nov2022 08:36:03.555] [modloading-worker-0/INFO] [com.tterrag.registrate.AbstractRegistrate/]: Detected new forge version, registering events reflectively.
[04Nov2022 08:36:03.556] [modloading-worker-0/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Failed to create mod instance. ModID: minetogether, class net.creeperhost.minetogether.forge.MineTogetherForge
java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:79) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:483) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:67) ~[javafmllanguage-1.18.2-40.1.84.jar%23263!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[fmlcore-1.18.2-40.1.84.jar%23262!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: java.lang.ExceptionInInitializerError
	at net.creeperhost.minetogether.MineTogetherClient.init(MineTogetherClient.java:40) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at net.creeperhost.minetogether.MineTogether.init(MineTogether.java:55) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at net.creeperhost.minetogether.forge.MineTogetherForge.<init>(MineTogetherForge.java:18) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67) ~[?:?]
	... 11 more
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.UUID.toString()" because "this.uuid" is null
	at net.creeperhost.minetogether.chat.ChatAuthImpl.<init>(ChatAuthImpl.java:32) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at net.creeperhost.minetogether.chat.MineTogetherChat.<clinit>(MineTogetherChat.java:44) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at net.creeperhost.minetogether.MineTogetherClient.init(MineTogetherClient.java:40) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at net.creeperhost.minetogether.MineTogether.init(MineTogether.java:55) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at net.creeperhost.minetogether.forge.MineTogetherForge.<init>(MineTogetherForge.java:18) ~[minetogether-forge-1.18.2-6.0.8.jar%23185!/:?]
	at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67) ~[?:?]
	... 11 more
```

### Version

`minetogether-forge-1.18.2-6.0.7.jar` and `minetogether-forge-1.18.2-6.0.8.jar` as well

### Minecraft Version

1.18.2

### ModLoader Version

Forge 40.1.84

### Additional Information

Related issues:
https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1597 (MT 6.0.7)
https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1611 (MT 6.0.8)
Edit: https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1679 (MT 6.0.8)

Thanks!

**Comment by Raidobw2** [comment]
Thx covers for fix

**Comment by covers1624** [comment]
This is fixed in MineTogether 6.0.10.

**Comment by mattarello03** [comment]
I also have the same problem with version 6.0.11 !
		minetogether-forge-1.18.2-6.0.11.jar              |MineTogether                  |minetogether                  |6.0.11              |ERROR     |Manifest: 31:e6:db:63:47:4a:6e:e0:0a:2c:11:d1:76:db:4e:82:ff:56:2d:29:93:d2:e5:02:bd:d3:bd:9d:27:47:a5:71
[crash-2022-11-13_09.46.08-client.txt](https://github.com/CreeperHost/MineTogether/files/9996785/crash-2022-11-13_09.46.08-client.txt)
